lib/locales/en/frontend/organizations.json

```
@@ -1,7 +1,6 @@
 {
   "all_organizations_title": "Organizations launching Hackathons and Challenges",
   "all_organizations_page_description": "Startups, Businesses, Universities, Investors, Tech Communities, Coding Schools and other Organizations launching hackathons on TAIKAI Hackathon Platform",
-  "launch_your_challenge": "Launch your Hackathon",
```

We are using this on `components/products/launch-challenge-modal.js`
```suggestion
  "launch_your_challenge": "Launch your Hackathon",
  ```